app/controllers/your-psp/post-toggle-worldpay-3ds-flex.controller.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const paths = require('../../paths')
+const { renderErrorView } = require('../../utils/response')
+const { ConnectorClient } = require('../../services/clients/connector.client')
+
+module.exports = async (req, res) => {
+    const connector = new ConnectorClient(process.env.CONNECTOR_URL)
```

Comment:
Am not too familiar with the implications of Node but in Java I would move this line outside of the function.

Reply:
I was just following the style of an existing file but I see we define it outside the function in other files, so I’ll do that here too.